contracts/SimpleSwap.sol

```diff
@@ -8,9 +8,9 @@ contract SimpleSwap {
   using SafeMath for uint;
 
   event Deposit(address depositor, uint amount);
-  event ChequeCashed(address indexed beneficiary, uint indexed serial, uint amount);
+  event ChequeCashed(address indexed beneficiary, uint indexed serial, uint payout, uint requestPayout);
   event ChequeSubmitted(address indexed beneficiary, uint indexed serial, uint amount, uint timeout);
-  event ChequeBounced(address indexed beneficiary, uint indexed serial, uint paid, uint bounced);
+  event ChequeBounced();
```

Comment:
Does it make sense to keep this event?

Reply:
yes, it is an easy way to see if a swap channel was not solved at some time.